### PR TITLE
docs/Developers/Review.rst: Add a note to use one ack per commit

### PR DESCRIPTION
**Pull request by gaurav1911**
This adds a note explaining that only one acknowledgement is needed per commit.

Fixes https://github.com/coala/coala/issues/3815

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by gaurav1911** [comment]
@jayvdb made changes as requested

File: docs/Developers/Review.rst [hunk 1]
```diff
@@ -72,6 +72,12 @@ The review process for coala is as follows:
     - ``reack commit_SHA`` in case the commit was acknowledged before, was
       rebased without conflicts and the rebase did not introduce logical
       problems.
+
+    .. note::
+
+        Only one acknowledge is needed per commit. There is no need to ack
```
Comment (Udayan12167) [inline docs/Developers/Review.rst]: I would reword this to say `Only one acknowledgement is needed per commit i.e. ack <commit-sha>.`

File: docs/Developers/Review.rst [hunk 2]
```diff
@@ -72,6 +72,11 @@ The review process for coala is as follows:
     - ``reack commit_SHA`` in case the commit was acknowledged before, was
       rebased without conflicts and the rebase did not introduce logical
       problems.
+
+    .. note::
+
+        Only one acknowledgment is needed per commit i.e. ack <commit-sha>.
```
Comment (blooops) [inline docs/Developers/Review.rst]: I think it would be better if the same placeholder was used for the commit-sha as in the rest of the document. Instead of <commit-sha> we could have commit_SHA (within the quotes ` ), as in the rest of the file to maintain uniformity

**Comment by gaurav1911** [comment]
@blooops made changes

**Comment by jayvdb** [comment]
ack 425e6df

**Comment by jayvdb** [comment]
needs rebasing

**Comment by fneu** [comment]
reack https://github.com/coala/coala/pull/3844/commits/c422efb51e578fa317c8f5b11a3c25a82e978134

**Comment by fneu** [comment]
@rultor merge

**Comment by fneu** [comment]
@gaurav1911 please rebase this on latest master, thanks :)

**Comment by gaurav1911** [comment]
@fneu rebased.

**Comment by fneu** [comment]
reack https://github.com/coala/coala/pull/3844/commits/5291cd91abef3b9a679f9d74a0934fe7d60e4523

**Comment by fneu** [comment]
@rultor merge
